pkg/filter/filter.go

```diff
@@ -0,0 +1,56 @@
+package filter
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/v3/pkg/label"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
+	capiutil "sigs.k8s.io/cluster-api/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/azure-admission-controller/pkg/release"
+)
+
+func IsCRProcessed(ctx context.Context, ctrlClient client.Client, logger micrologger.Logger, objectMeta metav1.Object) (bool, error) {
```

Comment:
```suggestion
func IsObjectReconciledByLegacyRelease(ctx context.Context, ctrlClient client.Client, logger micrologger.Logger, objectMeta metav1.Object) (bool, error) {
```

Minor nitpick: `IsCRProcessed` makes me think this function checks if the admission controller already processed given `object` while this is more about checking if the object is reconciled by legacy release. WDYT?

Reply:
:heavy_check_mark: